test/integrations/destinations/google_adwords_offline_conversions/network.ts

```diff
@@ -491,4 +491,121 @@ export const networkCallsData = [
       status: 200,
     },
   },
+  {
+    httpReq: {
```

Comment:
Please add more details to mocks like `description` check `braze` mocks for example or `campaign_manager`

Reply:
addressed